.renovaterc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    ":gitSignOff",
+    "group:monorepos",
+    "group:recommended",
+    ":pinAllExceptPeerDependencies",
```

Comment:
I was not able to find `:pinAllExceptPeerDependencies` preset...what does it do?

Reply:
I explained this in https://github.com/strongloop/loopback4-example-shopping/pull/93#issuecomment-489586795.

See https://renovatebot.com/docs/presets-default/#pinallexceptpeerdependencies

```json
{
  "packageRules": [
    {
      "packagePatterns": [
        "*"
      ],
      "rangeStrategy": "pin"
    },
    {
      "depTypeList": [
        "engines",
        "peerDependencies"
      ],
      "rangeStrategy": "auto"
    }
  ]
}
```